### PR TITLE
Feature : QuantStats

### PR DESCRIPTION
**Pull request by prithviraj-gotepatil-algobulls**
- changes required for new format
- changes in connection for pnl report and statistics report
- api for pnl report is updated
- statistics report generated using quantstats
- completed code simplification for pnl reports
- completed the cleanup of pnl and statistics reports cleanup

### CODE REVIEW

File: pyalgotrading/algobulls/api.py [hunk 1]
```diff
@@ -396,16 +396,19 @@ def get_reports(self, strategy_code: str, trading_type: TradingType, report_type
             `GET` v2/user/strategy/statstable       for Stats Table
             `GET` v2/user/strategy/orderhistory     Order History
         """
+        key = self.__get_key(strategy_code=strategy_code, trading_type=trading_type)
         if report_type is TradingReportType.PNL_TABLE:
-            endpoint = 'v2/user/strategy/pltable'
+            endpoint = 'v3/book/pl/data'
+            params = {'pageSize': 0, 'isPythonBuild': "true", 'strategyId': strategy_code}
         elif report_type is TradingReportType.STATS_TABLE:
```
Comment (pushpakdagade) [inline pyalgotrading/algobulls/api.py:403]: we no longer need this `else` block as we are computing stats directly.

File: pyalgotrading/algobulls/api.py [hunk 2]
```diff
@@ -396,16 +396,19 @@ def get_reports(self, strategy_code: str, trading_type: TradingType, report_type
...
         elif report_type is TradingReportType.ORDER_HISTORY:
             endpoint = 'v2/user/strategy/orderhistory'
+            params = {'key': key}
         else:
             raise NotImplementedError
 
-        key = self.__get_key(strategy_code=strategy_code, trading_type=trading_type)
-        params = {'key': key}
         response = self._send_request(endpoint=endpoint, params=params)
+        # print(f"delete this : {response}")
```
Comment (pushpakdagade) [inline pyalgotrading/algobulls/api.py:413]: cleanup

File: pyalgotrading/algobulls/api.py (hunk 1 above)
Comment (prithviraj-gotepatil-algobulls) [reply pyalgotrading/algobulls/api.py]: Done

File: pyalgotrading/algobulls/api.py (hunk 2 above)
Comment (prithviraj-gotepatil-algobulls) [reply pyalgotrading/algobulls/api.py]: Done

File: pyalgotrading/constants.py [hunk 3]
```diff
@@ -209,16 +209,5 @@ class DecisionConstants(Enum):
     EXIT_POSITION = 'EXIT_POSITION'
 
 
-class ActionConstants(Enum):
```
[the remark this answers is not in the document]
Comment (prithviraj-gotepatil-algobulls) [reply pyalgotrading/constants.py]: Done

**Review by pushpakdagade — APPROVED** [review]

**Pull request merged**